packages/vite/src/node/ssr/__tests__/ssrTransform.spec.ts

```diff
@@ -5,44 +5,45 @@ test('default import', async () => {
   expect(
     (
       await ssrTransform(
-        `import foo from 'vue';console.log(foo.bar)`,
+        `import foo from 'vue';\n` + `console.log(foo.bar)`,
         null,
         null
       )
     ).code
   ).toMatchInlineSnapshot(`
-    "const __vite_ssr_import_0__ = __vite_ssr_import__(\\"vue\\")
-    console.log(__vite_ssr_import_0__.default.bar)"
+    "const import_vue = __vite_ssr_import__(\\"vue\\");
```

Comment:
**question:** Are these changes only here in tests? Or also in the production code? If so, I thought they are intentionally named with `__` so they don't collide with other potential variables.

Reply:
[Sucrase](https://github.com/alangpierce/sucrase) does this same naming scheme, so I assume it's fine or they would've changed it. Although, they might scan all variables in the code to ensure no collisions.

If you can find a npm package that has a variable named this way, I'll consider it worth changing to avoid the collision. :)

Reply:
@patak-js ⬆️

Reply:
But it isn't just about a variable named in the package, this could conflict with the names defined in the user code. It is true that it will be extremely rare, because using a `_` is not conventional in JS. But for example: https://github.com/y52en/kirafan-image-merger/blob/77880e15d7dc6a7a72e3766d80215e9a02a2fda2/src/router.js, someone used it.
Maybe Sucrase may do a scan as you said? I would feel safer that we at least do a `__import_react` here.